applications/app/controllers/SurveyPageController.scala

```diff
@@ -26,8 +26,8 @@ class SurveyPageController(wsClient: WSClient) extends Controller with Execution
       Cached(defaultCacheDuration)(RevalidatableResult.Ok(views.html.survey.myDigest404(StaticPages.simpleSurveyStaticPageForId(request.path))))
    }
 
-  def renderWeekendReading404Page() = Action { implicit request =>
-      Cached(defaultCacheDuration)(RevalidatableResult.Ok(views.html.survey.weekendReading404(StaticPages.simpleSurveyStaticPageForId(request.path))))
+  def renderWeekendReadingPage() = Action { implicit request =>
+      Cached(defaultCacheDuration)(RevalidatableResult.Ok(views.html.survey.weekendReading(StaticPages.simpleSurveyStaticPageForId(request.path))))
```

Comment:
This is a really good point, thank you for raising and explaining  😄 

I will see if I can fix, or partially resolve this.